static/scss/partials/banners.scss

```diff
@@ -6,6 +6,10 @@
     ul {
         list-style-type: none;
     }
+    
+    .is-premium & {
```

Comment:
What does the `&` at the end of this class do?

Reply:
It's a [parent selector](https://sass-lang.com/documentation/style-rules/parent-selector), which is pure SCSS Magic™. It's the same as writing `.is-premium .recruitment-banner`.

Reply:
In this case, `.is-premium` is being added to the `<body>`, so it's influencing/cascading downstream to the banners.

Reply:
Ahhh I see! Gotcha